src/renderer/components/settings-electron.tsx

```diff
@@ -390,7 +428,7 @@ export const ElectronSettings = observer(
           break;
       }
 
-      if (version === appState.currentElectronVersion.version) {
+      if (this.state.activeVersions.has(appState.getVersionLockName(version))) {
```

Comment:
Can this implementation be refactored so that `AppState.getVersionLockName` isn't a public method? Feels like it's exposing too much internal implementation details of how active versions are tracked - more ideally it should just track the version numbers, not the lock name. I think maybe changing the implementation of `getActiveVersions` to return just the versions, not the lock names, would accomplish that.